src/polycubed/src/service_controller.cpp

```diff
@@ -284,5 +284,26 @@ bool ServiceController::parse_peer_name(const std::string &peer,
   return true;
 }
 
+void ServiceController::netlink_notification(int ifindex,
```

Comment:
In general, can we add a little comment about what functions are about?

Reply:
Sure,